### ISSUE TITLE
 reload value without restart (HUP signal): log-level or  separate log for new metric

### ISSUE THREAD

**Issue by ihard** [go-graphite/go-carbon#154]
As a suggestion to add a reload value without restart (HUP signal) - log-level.
You often need to see new metrics. They are only available in the debug mode.
Dump and start takes considerable time.
Or add the ability to output new metrics in a separate log.

**Comment by lomik** [comment]
Reload logging config without restart is too hard. You shouldn't expect this in the near future.
But ability to log new whisper files in separate log expected in next release.

**Comment by lomik** [comment]
Separate log for creates has been added in master. You can configure logging like this:
1. Remove old common.logfile and common.log-level options
2. Add this sections in config file:
```
[logging]
logger = ""
file = "/var/log/go-carbon/go-carbon.log"
level = "info"

[logging]
logger = "whisper:new"
file = "/var/log/go-carbon/new.log"
level = "debug"
```

**Comment by ihard** [comment]
Сool! Thank you.

**Comment by lomik** [comment]
Separate logging has been released in 0.10.0
